about.html

```diff
@@ -0,0 +1,117 @@
+            <span class="navbar-toggler-icon open-btn"></span>
+            <i class="fa-solid fa-x close-btn"></i>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                <a class="nav-link" href="./index.html">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#about">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#sponsor">Sponsor</a>
+              <li class="nav-item">
+                <a class="nav-link" href="#news">News</a>
+              </li>
+              <li class="nav-item">
+                <button class="join-btn">Join now</button>
+              </li>
+          </div>
+        </div>
+      </nav>
+  </header>
+
+  <section id="home" class="hero mb-4 bg-light text-dark">
+          <h1 class="intro-title text-center">Ruby on rails bootcamp 2023</h1>
+          <p class="lead mb-4 intro-p text-center">Welcome to our Ruby on Rails Bootcamp! This intensive program is designed to equip you with the knowledge and skills needed to build robust web applications using Ruby on Rails!
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur porro omnis sunt architecto officia commodi qui, perferendis voluptates accusamus mollitia enim eligendi. Blanditiis error sed ipsa non corporis a deleniti!
+            Lorem ipsum dolor, sit amet consectetur adipisicing elit. Repudiandae minima dignissimos pariatur, est cum laborum quas veritatis harum quaerat ducimus dolore deleniti quasi consequatur doloremque dolores nemo quod, tenetur beatae!
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni tenetur et nihil est totam quibusdam, laboriosam dolorem voluptatibus in omnis consectetur quisquam, fuga dolor deleniti sapiente! Doloribus placeat nulla nostrum?
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sit laboriosam veniam ullam suscipit quae? Non nisi quasi incidunt velit aut id quam voluptate ipsam assumenda, et eveniet esse culpa repudiandae!
+
+          </p>
+          <p class="text-dark text-center">Please contact us per Email for any further questions about the RUBY ON RAILS BOOTCAMP 2023!</p>
+          <a href="#" class="btn btn-outline-dark btn-lg text-center">khadaf3@gmail.com</a>
+  </section>
+
+  <section class=".brand" id="brand">
+    <h1 class="brand-title text-center">Ruby on rails bootcamp 2023</h1>
+    <hr class="orange-hr">
+    <p class="lead mb-4 text-center">Welcome to our Ruby on Rails Bootcamp! This intensive program is designed to equip you with the knowledge and skills needed to build robust web applications using Ruby on Rails!</p>
```

Comment:
Kindly align the text to the center with the appropriate padding and margin so as to have the same styling as what we have in the Behance template. 📦 🥺 
| Your Design | Figma Design |
| :---:   | :---: | 
|<img width="1336" alt="image" src="https://user-images.githubusercontent.com/73854999/225428410-245998ae-6235-4c78-af7a-cf359063be31.png"> |<img width="1063" alt="image" src="https://user-images.githubusercontent.com/73854999/225428512-513d23fa-c88c-4f55-9c21-55b4b955501c.png"> |